src/gui/editing.cpp

```diff
@@ -607,6 +591,796 @@ void FurnaceGUI::doPaste(PasteMode mode, int arg, bool readClipboard, String cli
   }
 }
 
+uint64_t convert_effect_openmpt_mod(char symbol, uint16_t val)
```

Comment:
convertEffectMPT_MOD (Yeah, let's use underscore here)

**Do not use unspecified chars** - these are signed on x86 and unsigned on ARM. Always append `signed` or `unsigned` to char UNLESS used for a C string.

Replace uint16_t with unsigned short

Reply:
Why you want unspecified length short instead of specified length uint16_t?

Reply:
Unspecified length short? Short is 16-bit in most compilers.

Now don't tell me you're using a particular compiler...

Reply:
I mean, if you are worried about sizes and signedness, you need to use stdint types. Because int, char, short, etc. sizes are not guaranteed, yet you perform bit operations on ints.

Reply:
The rest of the code uses non-_t types. Change to unsigned short for now.